dashboard/app/models/pd/application/principal_approval_application.rb

```diff
@@ -211,6 +228,7 @@ def additional_text_fields
         [:committed_to_diversity, TEXT_FIELDS[:other_please_explain], :committed_to_diversity_other],
         [:replace_which_course_csd, TEXT_FIELDS[:other_please_explain], :replace_which_course_csd_other],
         [:replace_which_course_csp, TEXT_FIELDS[:other_please_explain], :replace_which_course_csp_other],
+        [:replace_which_course_csa, TEXT_FIELDS[:other_please_explain], :replace_which_course_csa_other],
```

Comment:
would it be possible to provide a screenshot of where this set of changes shows up in the application?

Reply:
to be more specific, I'm asking about where the changes on the ruby side show up -- I think I can see where the changes on the jsx side show up.

Reply:
![image](https://user-images.githubusercontent.com/82416901/141029358-b03bd00f-7e05-4ea7-8468-9e7753b2d763.png)
here is the form, not sure what you mean by 'where they show up on the ruby side'?

Reply:
Haha, sorry, I was trying to ask where the ruby changes show up in the UI. Thanks!